bundles/org.openhab.binding.solarforecast/README.md

```diff
@@ -59,12 +59,20 @@ See [DateTime](#date-time) section for more information.
 |-----------------|---------|--------------------------------------------------------|-----------------|----------|----------|
 | resourceId      | text    | Resource Id of Solcast rooftop site                    | N/A             | yes      | no       |
 | refreshInterval | integer | Forecast Refresh Interval in minutes                   | 120             | yes      | no       |
+| refreshManual   | boolean | Manual Forecast Refresh                                | false           | no       | yes      |
```

Comment:
Perhaps setting `refreshInterval` to 0 could have the same effect, WDYT? It would be a "special value", but OTOH it would be simpler not to have contradicting parameters, e.g. `refreshManual` = true and `refreshInterval` = 120.

Reply:
Believe it or not I thought today about exactly the same idea. I'll commit the change but gimme some days for testing.